### PR TITLE
[localization] Reformat LaTeX source code.

### PR DESCRIPTION
**Pull request by jensmaurer**
Also add two missing periods and remove one instance
of superfluous whitespace.

### CODE REVIEW

File: source/locales.tex [hunk 1]
```diff
@@ -2573,64 +2266,62 @@
...
+\item
+For a signed integer value, the function \tcode{strtoll}.
+\item
+For an unsigned integer value, the function \tcode{strtoull}.
+\item
+For a \tcode{float} value, the function \tcode{strtof}.
+\item
+For a \tcode{double} value, the function \tcode{strtod}.
+\item
+For a \tcode{long double} value, the function \tcode{strtold}.
```
Comment (johelegp) [inline source/locales.tex:2308]: Why not indent each `\item` text by 2 spaces [as required](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines), or just remove the original blank lines between items?

Comment (jensmaurer) [reply source/locales.tex:2308]: I don't think we've made up our mind on `\item` source formatting just yet.

File: source/locales.tex [hunk 2]
```diff
@@ -4605,82 +4067,41 @@
...
-\tcode{char},
-rather than an array of
-\tcode{part},
-is specified for
-\tcode{pattern::field}
-purely for efficiency.
+An array of \tcode{char},
+rather than an array of \tcode{part},
+is specified for \tcode{pattern::field} purely for efficiency.
 \end{footnote}
```
Comment (johelegp) [inline source/locales.tex:4080]:
```suggestion
\end{footnote}.
```

File: source/locales.tex [hunk 3]
```diff
@@ -4605,82 +4067,41 @@
...
-such that the
-\tcode{part}
-value
-\tcode{static_cast<part>(p.field[i])}
-determines the
-$\tcode{i}^\text{th}$
-component of the format
+specified by a \tcode{pattern} value \tcode{p},
+such that the \tcode{part} value \tcode{static_cast<part>(p.field[i])}
+determines the $\tcode{i}^\text{th}$ component of the format
```
Comment (johelegp) [inline source/locales.tex:4075]:
```suggestion
\end{footnote}.
```

File: source/locales.tex [hunk 4]
```diff
@@ -2573,64 +2266,62 @@
...
-\item the most positive representable value,
+\item
+the most positive representable value,
 if the field to be converted to an unsigned integer type represents a value
 that cannot be represented in \tcode{val}.
-
-\item the converted value, otherwise.
+\item
+the converted value, otherwise.
 \end{itemize}
```
Comment (johelegp) [inline source/locales.tex:2325]: The capitalization here needs fixing.

Comment (jensmaurer) [reply source/locales.tex:2325]: Not in this patch.

File: source/locales.tex (hunk 1 above)
Comment (tkoeppe) [reply source/locales.tex:2308]: @zygoloid was ultimately against formatting that was difficult to relocate or regroup, so I think we may be moving away from that rule.

**Pull request merged**